apps/nextjs-website/react-components/components/Form/Form.tsx

```diff
@@ -327,53 +330,35 @@ const Form = ({
         *Campo obbligatorio
       </Typography>
       <Button
-        variant="contained"
+        variant='contained'
         sx={{ width: { md: 'auto', xs: '100%' }, zIndex: 4 }}
         onClick={handleSubmit}
         color={theme === 'dark' ? 'negative' : 'primary'}
       >
         Iscriviti
       </Button>
       <Typography
```

Comment:
```suggestion
{ notes && (
      <Typography
```
Add check for existence of notes.